src/core/ExpressionFormatter.ts

```diff
@@ -208,11 +208,18 @@ export default class ExpressionFormatter {
         return this.formatCaseStart(token);
       case TokenType.RESERVED_CASE_END:
         return this.formatCaseEnd(token);
-      case TokenType.PARAMETER:
+      case TokenType.NAMED_PARAMETER:
+      case TokenType.QUOTED_PARAMETER:
+      case TokenType.INDEXED_PARAMETER:
+      case TokenType.POSITIONAL_PARAMETER:
         return this.formatParameter(token);
+      case TokenType.COMMA:
+        return this.formatComma(token);
+      case TokenType.DELIMITER:
```

Comment:
This case never happens. All DELIMITER tokens gets consumed by parser and never reach ExpressionFormatter.

Reply:
That what I thought but I left it in as a just it case to make sure it doesn't trigger the default case

Reply:
I think here we do want to trigger the default case in case any semicolon slips through, so we'd get a clear error as soon as possible instead of failing silently in some harder to diagnose way.